src/Hintman/Suggest/TrailingNewline.hs

```diff
@@ -0,0 +1,19 @@
+module Hintman.Suggest.TrailingNewline
+       ( suggestNoTrailingNewline
+       ) where
+
+import Hintman (LineNumber)
+
+import qualified Data.Text as T
+
+
+suggestNoTrailingNewline :: (FilePath, Text) -> Maybe (FilePath, LineNumber, Text)
+suggestNoTrailingNewline (path, body) = (generateResult . reverse . dropWhile isNewline . reverse) splitBody
+    where
+        splitBody :: [Text]
+        splitBody = T.lines body
+        generateResult :: [Text] -> Maybe (FilePath, LineNumber, Text)
+        generateResult newBody | length newBody == length splitBody = Nothing
```

Comment:
`splitBody` gets passed to `generateResult`, so then this line compares `length splitBody` to `length splitBody`, which will always be equal. So this function always returns `Nothing` with the way it's currently called, unless I'm mistaken :thinking:

Reply:
Nevermind this, it works